services/datamanager/builtin/config.go

```diff
@@ -43,12 +43,13 @@ type Config struct {
 	DeleteEveryNthWhenDiskFull  int   `json:"delete_every_nth_when_disk_full"`
 	MaximumCaptureFileSizeBytes int64 `json:"maximum_capture_file_size_bytes"`
 	// Sync
-	AdditionalSyncPaths    []string `json:"additional_sync_paths"`
-	FileLastModifiedMillis int      `json:"file_last_modified_millis"`
-	MaximumNumSyncThreads  int      `json:"maximum_num_sync_threads"`
-	ScheduledSyncDisabled  bool     `json:"sync_disabled"`
-	SelectiveSyncerName    string   `json:"selective_syncer_name"`
-	SyncIntervalMins       float64  `json:"sync_interval_mins"`
+	AdditionalSyncPaths    []string             `json:"additional_sync_paths"`
+	FileLastModifiedMillis int                  `json:"file_last_modified_millis"`
+	MaximumNumSyncThreads  int                  `json:"maximum_num_sync_threads"`
+	ScheduledSyncDisabled  bool                 `json:"sync_disabled"`
+	SelectiveSyncerName    string               `json:"selective_syncer_name"`
+	SyncIntervalMins       float64              `json:"sync_interval_mins"`
+	MongoConfig            *capture.MongoConfig `json:"mongo_config"`
```

Comment:
Thoughts on naming this something like `capture_mongo_config` or `mongo_capture_config`? To make it clear that the data is being pushed to Mongo on the capture interval(s). Also, mind moving this to the section above under the comment `// Capture`?

Reply:
good call, will do

Reply:
done